ai_services/anomaly_detection/data_preprocessing_examples/oci_data_flow_based_examples/example_code/sharding.py

```diff
@@ -27,13 +27,13 @@ def sharding(df, **kwargs):
     Return:
         partitions of the original dataframe in CSV format
     """
-    partition_size = int(kwargs["columnNum"])
+    partition_size = int(kwargs["columnNum"]) if "columnNum" in kwargs else 300
```

Comment:
[Minor]
```suggestion
    partition_size = int(kwargs["columnNum"].get("300"))
```
Similar comment for other default values from dictionaries.

Reply:
I tested with my local Python3
```
>>> kwargs = dict()
>>> kwargs["columnNum"].get("300")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 'columnNum'
```
Looks like it's not working as expected